pandas/tests/io/test_pickle.py

```diff
@@ -376,6 +376,40 @@ def test_read_infer(self, ext, get_random_path):
 
             tm.assert_frame_equal(df, df2)
 
+    @pytest.mark.parametrize("cruft", ["__MACOSX/", ".DS_STORE"])
+    def test_load_zip_with_hidden_folders(self, cruft, get_random_path):
+        """Test loading .zip files that have extraneous metadata in hidden folders. """
+        base = get_random_path
+        path1 = base + ".raw"
+        path2 = base + ".zip"
+        dummy = base + ".dummy"
```

Comment:
pandas prefers f-strings: f"{base}.raw"

Reply:
Done, I didn't go as far as replacing it for all other tests in this file